addons/website_mail_channel/views/snippets/s_channel.xml

```diff
@@ -32,7 +32,7 @@
                     <!-- 'we-button' added programmatically with DB data -->
                 </we-select>
                 <we-button class="fa fa-fw fa-plus" title="Create a public discussion group in your backend"
-                           data-create-channel="" data-no-preview="true"/>
+                           data-create-channel="" data-no-preview="true" data-name="create_mail_channel_opt"/>
```

Comment:
As this is a fix, existing users on the saas will receive the JS update but the not the XML one. That means your fix, if not working (as non critical), should at least not make the user code crash. Here your JS code is requesting this widget by name... but if the XML is not updated it won't find any widget with that name, hence `widget.$el.click()` will crash. As a fix, you can tweak the `_renderCustomXML` to search for that button in the received `uiFragment` and add it its `data-name` (and we'll remove that code once your fix reaches master, you can add a comment "TODO remove this part in master" next to it).